source/website/website.js

```diff
@@ -657,9 +657,12 @@ export class Website {
     // });
     // AddSeparator(this.toolbar, ['only_full_width', 'only_on_model']);
     // AddButton(this.toolbar, 'snapshot', 'Create snapshot', ['only_full_width', 'only_on_model'], () => {
-    AddButton(this.toolbar, 'snapshot', 'Create snapshot', ['only_full_width'], () => {
-      ShowSnapshotDialog(this.viewer);
-    });
+
+    if (this.parameters.options?.button?.snapshot) {
+      AddButton(this.toolbar, 'snapshot', 'Create snapshot', ['only_full_width'], () => {
```

Comment:
you can do 
callParams = [
      this.toolbar,
      'snapshot', 
      'Create snapshot',
       ['only_full_width'], () => {
        ShowSnapshotDialog(this.viewer);
      }
   ];
   AddButton(...callParams)